packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -326,7 +384,131 @@ export class DetailsHeader extends BaseComponent<IDetailsHeaderProps, IDetailsHe
 
   /** Set focus to the active thing in the focus area. */
   public focus(): boolean {
-    return Boolean(this._root.current && this._root.current.focus());
+    return Boolean(this._root.value && this._root.value.focus());
+  }
+
+  private _getHeaderDragDropOptions(): IDragDropOptions {
+    const dataContext: IDragDropContext = { data: this, index: 1 };
+    const options = {
+      selectionIndex: 1,
+      context: { data: dataContext, index: 0 },
+      canDrag: () => false,
+      canDrop: () => true,
+      onDragStart: () => undefined,
+      updateDropState: this._updateDroppingState,
+      onDrop: this._dragDropColumnEvents!.onDrop,
+      onDragEnd: () => undefined,
+      onDragOver: this._dragDropColumnEvents!.onDragOver
+    };
+    return options;
+  }
+
+  private _updateDroppingState(newValue: boolean, event: DragEvent): void {
+    // TODO - Handle CSS changes
+  }
+
+  private _getDragDropColumnEvents(): IDragDropEvents {

```

Comment:
>private _getDragDropColumnEvents(): IDragDropEvents [](start = 2, length = 51)

Why is this required? We can do whatever we are doing here inside IDragDropOptions.